appveyor.yml

```diff
@@ -1,33 +1,45 @@
-build: false
+# https://ci.appveyor.com/project/goanpeca/pyswmm
+
+matrix:
+  fast_finish: true
+
+branches:
+  only:
+    - master 
 
 environment:
+  global:
+    # Used by qthelpers to close widgets after a defined time
+    TEST_CI: True
+    # Environment variables used by astropy helpers
+    PYTHON: "C:\\conda"
+    CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\ci-helpers\\appveyor\\windows_sdk.cmd"
+    PYTHON_ARCH: "64" # needs to be set for CMD_IN_ENV to succeed. If a mix
```

Comment:
64-bit swmm is available but remains untested.  Once we get CI for the OWA SWMM Repo and testing, we should have a better idea on how the results compare between 32 and 64 bit.  In the mean time, Go ahead and merge this PR